src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs

```diff
@@ -357,7 +357,7 @@ public async Task UpdatePullRequestAsync(string pullRequestUri, PullRequest pull
         /// <param name="pullRequestUrl">Uri of pull request to merge</param>
         /// <param name="parameters">Settings for merge</param>
         /// <returns></returns>
-        public async Task MergePullRequestAsync(string pullRequestUrl, MergePullRequestParameters parameters)
+        public virtual async Task MergePullRequestAsync(string pullRequestUrl, MergePullRequestParameters parameters)
```

Comment:
Why was this made virtual? I don't see it overridden anywhere.  The "GitHubClient" really shouldn't require any virtual methods....

Reply:
It looks like a _lot_ of virtual methods got added.

Reply:
I think it was refactor residue. I redid this.